generators/cucumber/templates/env.rb

```diff
@@ -37,4 +37,32 @@
     log.info "SAUCE SERVER LOG URL: #{Capybara::Settings.sauce_resource_path('selenium-server.log')}"
     Capybara::Settings.update_sauce_job_status(passed: DataStorage.extract('sauce', :status))
   end
+end
+
+# BrowserStack settings
+
+url = settings.bs_url
+
+capabilities = Selenium::WebDriver::Remote::Capabilities.new
+
+capabilities['project'] = settings.bs_build
+capabilities['build']   = settings.bs_project
+if settings.bs_mobile.true?
+  capabilities['os'] = settings.bs_os_name
+  capabilities['os_version'] = settings.bs_os_version
+else
+  capabilities['platform'] = settings.bs_m_platform
+end
+
+capabilities['browser'] = settings.bs_browser_name
+capabilities['browser_version'] = settings.bs_browser_version
+
+browser = Selenium::WebDriver.for(:remote, :url => url, :desired_capabilities => capabilities)
+
+Before do |scenario|
```

Comment:
I am not sure it is really required, because it is too expensive for us to close and open browser after each scenario. Instead of that we use clearing of cookies